content/tsColinPrecip.Rmd

```diff
@@ -44,12 +47,20 @@ Function to retrieve precip data using [`geoknife`](https://github.com/USGS-R/ge
 ```{r getPrecip, message=FALSE}
 getPrecip <- function(states, startDate, endDate){
   
-  wg_s <- webgeom(geom = 'derivative:US_Counties', attribute = 'STATE')
-  wg_c <- webgeom(geom = 'derivative:US_Counties', attribute = 'COUNTY')
+  # fips webgeom
   wg_f <- webgeom(geom = 'derivative:US_Counties', attribute = 'FIPS')
-  county_info <- data.frame(state = query(wg_s, 'values'), county = query(wg_c, 'values'), 
-                            fips = query(wg_f, 'values'), stringsAsFactors = FALSE) %>% 
-    unique() 
+  
+  # use fips data from US Census Bureau (found in Data.gov catalog:
+  # https://catalog.data.gov/dataset/fips-county-code-look-up-tool)
+  fips_lookup <- read.csv('http://www2.census.gov/geo/docs/reference/codes/files/national_county.txt', 
```

Comment:
I would suggest doing this differently, because there is a chance that the fips you use for mapping don't match the fips you get from that file. 

you are plotting the counties with `maps::map('county')`, so using the fips from that package makes sense to me:

```r
maps::county.fips
     fips                            polyname
1     1001                     alabama,autauga
2     1003                     alabama,baldwin
3     1005                     alabama,barbour
4     1007                        alabama,bibb
5     1009                      alabama,blount
6     1011                     alabama,bullock
7     1013                      alabama,butler
....
```